grpc/src/main/java/com/linecorp/armeria/server/grpc/ArmeriaServerCall.java

```diff
@@ -193,6 +194,16 @@
         this.compressorRegistry = requireNonNull(compressorRegistry, "compressorRegistry");
         clientAcceptEncoding =
                 Strings.emptyToNull(clientHeaders.get(GrpcHeaderNames.GRPC_ACCEPT_ENCODING));
+        if (autoCompression && clientAcceptEncoding != null) {
+            final List<String> acceptedEncodingsList =
+                    ACCEPT_ENCODING_SPLITTER.splitToList(clientAcceptEncoding);
```

Comment:
Could use `split()` instead of `splitToList()` because we don't need a list here and an iterable is potentially better in terms of overhead.